reconcile/utils/terraform_client.py

```diff
@@ -14,6 +14,7 @@
 from threading import Lock
 from typing import (
     Any,
+    Callable,
```

Comment:
Same as above: `from collections.abc import Callable`

Reply:
done.

Reply:
I think the change wasn't pushed. This is still imported `from typing`

Reply:
oops! fixed this now.